client/src/pages/Login/login.css

```diff
@@ -0,0 +1,35 @@
+.wrapp{
+    min-width: 370px;
+    height: 100vh;
+}
+
+.content{
+    max-width: 370px;
+    margin: auto auto;
+}
+
+.sidebar{
+    background-color: #F4F4F4;
+}
+
+.recapcha{
+    color: #9A9FA5;
+}
+
+@media screen and (min-width: 1024px) {
+    .sidebar{
+        width: 40%;
+    }
+    .content{
+        width: 60%;
+    }
+}
+
+@media screen and (min-width: 1240px) {
+    .sidebar{
+        width: 30%;
+    }
+    .content{
+        width: 70%;
+    }
+}
```

Comment:
I think we should use variable for width. And when the breakpoint change, just change the variable